### DISCUSSION TITLE
DocumentDB change stream event

### DISCUSSION THREAD

**Discussion by MajdSoufan** [DefinitelyTyped/DefinitelyTyped#65403] [Request a new @types package]
On February 28th of this year, AWS started supporting DocumentDB as an event source to AWS Lambda.
Can we have a type for that event in aws-lambda for typescript?
The type of the event can be extracted from the example in this AWS developer guide
https://docs.aws.amazon.com/lambda/latest/dg/with-documentdb.html

**Comment by bneigher** [comment]
Would also love this
